coalib/__init__.py

```diff
@@ -1,3 +1,10 @@
+"""
+The coalib package is a collection of various subpackages regarding writing
+and editing bears. Various other packages such as formatting and settings are
```

Comment:
also executing, the main parts are about running bears and interacting with the user